src/gengowatcher/ui_textual.py

```
@@ -600,6 +600,12 @@ def refresh_metrics(self) -> None:
             else:
                 elapsed_hours = 1.0  # Default to 1 hour if no session start
             rate = found / elapsed_hours
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)